pkg/controller/configurationpolicy/configurationpolicy_utils.go

```diff
@@ -68,8 +69,29 @@ func updateRelatedObjectsStatus(list []policyv1.RelatedObject,
 	return list
 }
 
+func checkGenericObjWithSort(mergedObj interface{}, oldObj interface{}) (areEqual bool) {
```

Comment:
With this name, it's not obvious what the function "checks". In a condition like `if checkGenericObjWithSort(foo, bar) {...}` I'd need to look up what this function does... the named return helps, but it still makes me thing more than I want.

Can we call this something more like `equalObjWithSort`? I think code like `if equalObjWithSort(foo, bar) {...}` would be more obvious.

Reply:
sure, I will change it. thanks